cnames_active.js

```diff
@@ -138,6 +138,7 @@ var cnames_active = {
   "akase": "cedmax.github.io/akase", // noCF? (don´t add this in a new PR)
   "akhyar": "akhyar.pages.dev",
   "aksara": "sajenid.github.io/aksara.js",
+  "aksharamukha": "paramsiddharth.github.io",
```

Comment:
```suggestion
  "aksharamukha": "paramsiddharth.github.io/aksharamukha.js",
```
I assume you're wanting to target https://github.com/paramsiddharth/aksharamukha.js

Reply:
Yes, that's correct. Pardon.